code-challenges.test.js

```diff
@@ -26,34 +26,149 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 // a) Create a test with expect statements for each of the variables provided.
 
 const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
+
+describe("divisableByThree", () => {
```

Comment:
I would recommend condensing these tests into one describe and it block and just having multiple expect statements to test each variable.